typings/index.d.ts

```diff
@@ -130,7 +130,14 @@ export interface DateTimePickerProps {
    *
    * Default is 'date'
    */
-  mode?: "date" | "time" | "datetime";
+  mode?: 'date' | 'time' | 'datetime' | 'countdown';
+
+  /**
+   *  Display for the picker
+   * 
+   *  switches the Views for the picker
+   */
+   display?: Display | IOSDisplay
```

Comment:
Can we remove the `display` definition here? It should be inferred from the changes below 👍